### DISCUSSION TITLE
Where is The Simple Vanilla Template?

### DISCUSSION THREAD

**Discussion by KiddoV** [wailsapp/wails#1377] [Q&A] [answered]
Not sure how the vanilla template got update, but how do I get a simple vanilla template without the ``npm``?
``wails init -n project -t vanilla``. I just need a pure JS template.
Vanilla JS supposed to not having anything (frameworks) right? excluding ``vite``?

**Comment by leaanthony** [comment, marked as answer]
We still support it but it looks like it accidentally got removed as part of the internal template generation work. I'll look at reading it to master today.

**Reply by KiddoV** [reply]
Thanks @leaanthony

**Reply by leaanthony** [reply]
How did you go with this?

**Reply by KiddoV** [reply]
Just make my own template I guess... while waiting for you to fix the vanilla template on master.

**Reply by leaanthony** [reply]
https://github.com/wailsapp/wails/commit/6224d3aa82649e311557418273d5693baeb55ec7

**Reply by KiddoV** [reply]
How do I test this commit? 
I am still seeing ``npm`` and ``vite`` when using ``wails init -n myproject -t vanilla``

**Reply by leaanthony** [reply]
It's there. Did you rebuild the CLI?

Edit: It's now called `plain`

**Reply by KiddoV** [reply]
```console
> wails init -n myproject -t plain
Wails CLI v2.0.0-beta.36


Initialising Project 'myproject'
--------------------------------


ERROR: repository not found
```
Hmm.. how do you "rebuild" the CLI?
